pgcli/pgcompleter.py

```diff
@@ -138,7 +138,6 @@ def escaped_names(self, names):
         return [self.escape_name(name) for name in names]
 
     def extend_database_names(self, databases):
-        databases = self.escaped_names(databases)
```

Comment:
Is the problem here that we escape database names, or that `escape_name` does not check for the name being already quoted?

Reply:
I think we shouldn't be escaping database names because having MiXedCasE database names are valid in Postgres. So `\c MiXedCasE` is a perfectly legal command and it works.